tasks/release.rb

```diff
@@ -343,3 +343,21 @@ def rc?
     puts ERB.new(template, trim_mode: "<>").result(binding)
   end
 end
+
+# Check that various utilities are working so that when we push gems it won't fail
+task :check_utilities do
+  # try getting an OTP from the yubikey for RubyGems
+  sh "ykman oath accounts code -s rubygems.org"
+
+  # try getting an OTP from the yubikey for npm
+  sh "ykman oath accounts code -s npmjs.com"
```

Comment:
I release from a dev container and there is no way `ykman` can read the hardware token in my mac, so I can't have this check happening otherwise I'd not be able to release.

Reply:
Maybe we could run this test only if `ykman` is installed, which presumably it wouldn't be on your dev containers. Would that work for you?